features/step_definitions/basic_steps.rb

```diff
@@ -1,7 +1,34 @@
-When("I visit index page") do
+When("I visit the index page") do
   visit root_path
 end
 
+Then("show me the page") do
+  save_and_open_page
+end
+
 Then("I should see {string}") do |content|
   expect(page).to have_content content
 end
+
+Then("I click {string}") do |link|
+  click_link_or_button link
```

Comment:
You can just have click_link here

Reply:
Chnaged to click_link_or_button element so we can use it for both links and buttons everywhere

Reply:
Good question, is it family channel, family tree or is it the user's welcome page for navigating through family channels to get to the family trees? Or are we doing this iteratively? 
We'll remove flash messages from the index but keep it on the family page.

Reply:
To answer your question Sophie, Family page is the page where we display the family tree after adding family members (but this is the separate story)

Reply:
We display flash message on family page because this is where we navigate after successfull signup/login.